scripts/thresholds.sh

```diff
@@ -2,6 +2,6 @@
 set -e
 
 export LOWER_PYLINT_THRESHOLD=1000
-export UPPER_PYLINT_THRESHOLD=2400
+export UPPER_PYLINT_THRESHOLD=7004
```

Comment:
It looks like the new run had 7005; you may want to give yourself a little padding (something like 7025 or 7050) to allow for minor fluctuation between runs.